src/styles/Header.less

```diff
@@ -0,0 +1,30 @@
+.header {
+    height: 5rem;
+    margin-top: 1.4rem;
+    margin-bottom: 1.4rem;
+    display: flex;
+    justify-content: space-between;
+    font-size: 1.24rem;
+
+    &--rk_logo {
+        height: 2.5rem;
+        vertical-align: middle;
+        cursor: pointer;
+        text-decoration: none;
+    }
+
+    &--logo {
+        font-weight: bold;
+        white-space: nowrap;
+        color: black;
```

Comment:
Use color variables in separate file?

Reply:
Legger til color.less nå slik det er gjort i repoet til rodekors.